2-mandatory/4-stocks.js

```diff
@@ -34,7 +34,17 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Functions can help with this!
 */
 function getAveragePrices(closingPricesForAllStocks) {
-    // TODO
+
+    let agv = 0;
+    let avgPrices = [];
+    for (let allPrices of closingPricesForAllStocks) {
+        let sum = 0;
+        for (let closingPrices of allPrices)
+            sum += closingPrices;
+        avg = sum / allPrices.length;
```

Comment:
shouldn't be an issue here, but take care when doing division with things that could be 0; it will crash